aleph_alpha_client/aleph_alpha_client.py

```diff
@@ -65,6 +65,7 @@
     SteeringConceptCreationResponse,
 )
 from aleph_alpha_client.version import MIN_API_VERSION, user_agent_headers
+from aleph_alpha_client.translate import TranslationRequest, TranslationResponse
```

Comment:
Given that all the other module names are nouns, I would prefer `translation` here as well.